app/views/design-history/beta/iterating-the-task-list-and-adding-the-project-information-tab.html

```diff
@@ -0,0 +1,68 @@
+    acting on some of the feedback we got during the user research.</p>
+      <p class="govuk-body">This included:</p>
+      <ul class="govuk-list govuk-list--bullet">
+        <li> Status tasks to reflect a task's status - these now show if a task
+          is yet to set, is in progress or is completed  </li>
+        <li> Explicitly refer to Regional Delivery Officers in handover task questions
+          - this should improve clarity for users, avoiding confusion with Operational 
+          Delivery Team Delivery Officers (our users)</li>
+      </ul>
+    <h2 class="govuk-heading-l">Name changes throughout the division</h2>
+    <p class="govuk-body">There have been multiple changes to areas of the department, 
+      the group and the division we and our users belong to.</p>
+    <p class="govuk-body">There have also been changes to the name of the team our users 
+      work in, and their job titles.</p>
+    <p class="govuk-body">This has meant we've made changes in the prototype to reflect 
+      this.</p>
+    <p class="govuk-body">The team our users are part of has changes from Operational Delivery
+      Team to Regional Casework Services.</p>
+    <p class="govuk-body">Our users' job titles have changed from Delivery officers to Caseworkers.</p>
+    <h2 class="govuk-heading-l">Preparing versions of the handover process for user research</h2>
+    <p class="govuk-body">To help us continue testing and iterating the task list and the actions
+      users take to complete those tasks, we created an alternative version of the handover task.</p>
+    <p class="govuk-body">This removed unnecessary questions and listed all the relevant questions
+      on one page, rather than one question per page.</p>
+    <h2 class="govuk-heading-l">Preparing the project information tab for user research</h2>
+    <p class="govuk-body">Following discussions with some of the caseworkers, we were able to get more 
+      information about what they needed from a project information tab.</p>
+    <p class="govuk-body">The idea of a project information tab was something that had been included
+      in the Figma screens.</p>
+    <p class="govuk-body">This contained all the background information about the project, including
+      contact details, SharePoint file locations and school type.</p>
+    <p class="govuk-body">We wanted to start investigating more about how useful that was, and what
+      information in it was most useful.</p>
+    <h2 class="govuk-heading-l">Document journeys</h2>
+    <p class="govuk-body">We also built journeys for checking and clearing legal documents and other
+      files needed to progress the conversion.</p>
+    <p class="govuk-body">This included:</p>
+       <ul class="govuk-list govuk-list--bullet">
+        <li>Local authority questionnaire</li>
+        <li>Conversion grant</li>
+        <li>Non-legal documents</li>
+      </ul>
+  </div>
+</div>
+
+
+{% endblock %}
```

Comment:
All files should end with an empty line so we know its the end of the code and not an error